File: Backend/API/Controllers/Models/CreateFreelancerInput.cs
namespace Backend.API.Controllers.Models
{
    public class CreateFreelancerInput
    {
        public string Name { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string Phone { get; set; }
        public List<string> Skills { get; set; }
        public List<double> Ratings { get; set; }


        

        public CreateFreelancerInput(string name, string email, string password, string phone, List<string> skills, List<double> ratings)
        {
            Name = name;
            Email = email;
            Password = password;
            Phone = phone;
            Skills = skills;
            Ratings = ratings;
            
        }
    }
}

File: frontend/src/models/CreateFreelancerInput.ts
import Skill from "./Skill";

export default class CreateFreelancerInput {
    Email: string;
    Password: string;
    Name: string;
    Phone: string;
    Skills: string[];
    Ratings: number[];

    constructor(email: string, password: string, name: string, phone: string, skills: string[], ratings: number[]) {
        this.Email = email;
        this.Password = password;
        this.Name = name;
        this.Phone = phone;
        this.Skills = skills;
        this.Ratings = ratings;
    }
};